### PR TITLE
Address #92, update folder structure for test files.

### PR DESCRIPTION
**Pull request by mikelax**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --reporter spec"
```
Comment (arthurvr) [inline package.json]: wasn't `spec` the default?

Comment (mikelax) [inline package.json]: It is, I was testing with the --check-leaks option and by default I always add both. I removed check-leaks but just left in reporter. Makes it easy to change if needed. I can remove if you want.

Comment (kevva) [inline package.json]: Yeah, remove it.

**Comment by kevva** [comment]
Could you squash your commits too? :)

**Comment by mikelax** [comment]
keeping it clean, nice.  should be all set.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**